.changeset/real-ends-switch.md

```diff
@@ -0,0 +1,9 @@
+---
+'astro': minor
+---
+
+Enables CLI shortcuts for `astro preview`:
+
+- `o + enter`: opens the site in your browser
+- `q + enter`: quits the dev server
```

Comment:
```suggestion
- `q + enter`: quit the preview
```
Is this more accurate to say? The "dev" server isn't running if you're previewing?

And, aren't these shortcuts usually listed as the action available for you to perform (i.e. press this key to "quit"... not "quits"?) This is how we list them in the [CLI reference](https://docs.astro.build/en/reference/cli-reference/). If this is changed, change the other verbs, too!